Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,32 @@
-function setAlarm() {}
+const input = document.querySelector("input");
+let time = 0;
+let timerId;
+
+function setAlarm() {
+  time = input.valueAsNumber;
+  changeDisplayTime(time);
+  timerId = setInterval(deduction, 1000);
+}
+
+function deduction() {
+  --time;
+  changeDisplayTime(time);
+  if (time == 0) {
+    clearInterval(timerId);
+    playAlarm();
+    document.body.style.background = "#ff4d4d";
+  }
+}
+
+function changeDisplayTime(time) {
```

Comment:
Good job extracting a function here to reduce duplication between the initial "set alarm" and the updates every second!